src/Microsoft.AspNet.Mvc.Core/ModelBinding/ByteArrayModelBinder.cs

```diff
@@ -39,7 +39,10 @@ public class ByteArrayModelBinder : IModelBinder
             try
             {
                 var model = Convert.FromBase64String(value);
-                return new ModelBindingResult(model, bindingContext.ModelName, isModelSet: true);
+                return new ModelBindingResult(
+                    model,
+                    bindingContext.ModelName,
+                    isModelSet: true);
```

Comment:
Should we be creating a node here? This seems to imply that a `byte[]` property can't have validators.